tests/test_resolve.py

```
@@ -2095,13 +2438,384 @@ def _commit(status, message):
+    - IN PLACE, same uncommitted tree: SILENT. The re-drive reads the main checkout's
+      working tree, which is exactly where `cwd=project` put the correction. The
+      recorded mount is identical on all three rows precisely so it cannot be what
+      separates them.
+
+    Ablation: drop the `_redrive_reads_the_upstream_artifacts` conjunct from the gate
+    and row 2 reddens on `assert True is False`; make `stories_reach_the_redrive` return
+    False for the in-place leg and row 3 reddens; delete the whole `if` and row 1
+    reddens on `assert [] != []`. Every row alone is satisfied by a wrong fix.
+    """
+    intent = CORRECTED_INTENT if committed_matches else WEDGED_INTENT
+    run_dir, state, sentinel = _sentinel_run(
```

Ruff (0.16.2)

[warning] Unpacked variable `state` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)